src/containers/ZoomControls.js

```diff
@@ -33,13 +33,20 @@ const mapDispatchToProps = { updateViewport: actions.updateViewport };
  * ListItem: {paddingBottom: number, paddingTop: number}}}
  */
 const styles = theme => ({
+  divider: {
+    borderRight: '1px solid #808080',
```

Comment:
I couldn't find a matching color in the theme and the `divider` color in the palette is way too light.  Is there something we should be using here from the theme?

Reply:
I think we may just need to come back to this when we do more theme things.